### PR TITLE
Add how to send and receive context

### PR DESCRIPTION
**Pull request by feeblefakie**
This PR adds some descriptions about how to send and receive context in ScalarDL Contracts and Functions.

### CODE REVIEW

**Review by supl — APPROVED** [review]
LGTM.
Thank you!

File: docs/how-to-write-function.md [hunk 1]
```diff
@@ -108,6 +108,17 @@ A `functionArgument` is a runtime argument for the Function specified by the req
 
 `contractArgument` and `contractProperties` are the corresponding contract's argument and properties. See [How to Write a Contract](how-to-write-contract.md) to understand what they are.
 
+### Receive information from Contracts
+
+In non-deprecated Functions like `JacksonBasedFunction`, you can receive some information from Contracts by calling `T getContractContext()`.
+Note that the return value can be null if nothing has been set by Contracts and the return value type `T` will be decided based on a base Function class that you use.
```
Comment (josh-wong) [inline docs/how-to-write-function.md:114]: Revised to reduce passive voice.

```suggestion
Note that the return value can be null if Contracts has nothing set and the base Function class that you use will decide the return value type `T`.
```

File: docs/how-to-write-function.md [hunk 2]
```diff
@@ -108,6 +108,17 @@ A `functionArgument` is a runtime argument for the Function specified by the req
...
+See [this](./how-to-write-contract.md#send-information-to-functions) for how to send information to Functions from Contracts.
+
```
Comment (josh-wong) [inline docs/how-to-write-function.md:116]: Revised to specify link name.

```suggestion
For details on how to send information to Functions from Contracts, see [Send information to Functions](./how-to-write-contract.md#send-information-to-functions).
```

**Review by josh-wong — CHANGES_REQUESTED** [review]
I made a few suggestions. PTAL!

**Comment by feeblefakie** [comment]
@josh-wong Thanks! Applied all the suggestions.

**Review by josh-wong — APPROVED** [review]
LGTM!🚀

**Review by jnmt — APPROVED** [review]
LGTM! Thank you! I don't think this is the scope of this PR, but it would be nice if we could add a use case or a concrete example that uses this feature so that users get hints about when they should use this in their development.

**Review by reddikih — APPROVED** [review]
LGTM! Thank you.

> LGTM! Thank you! I don't think this is the scope of this PR, but it would be nice if we could add a use case or a concrete example that uses this feature so that users get hints about when they should use this in their development.

I would like to make a sample code and a blog post to show the usage of this feature.

**Pull request merged**